Dockerfile

```diff
@@ -1,70 +1,79 @@
-FROM python:3.8.6-slim AS base
-
-ENV LANG C.UTF-8
-ENV LC_ALL C.UTF-8
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONFAULTHANDLER 1
-ENV ABS_DIR="/abs"
-ENV PIPENV_VER="2020.11.15"
-RUN mkdir $ABS_DIR
+FROM quay.io/giantswarm/alpine:3.12 AS binaries
+
+ARG HELM_VER="3.3.4"
+ARG KUBECTL_VER="1.18.9"
+ARG CT_VER="3.1.1"
+ARG APPTESTCTL_VER="0.5.2"
+
+RUN apk add --no-cache ca-certificates curl \
+    && mkdir -p /binaries \
+    && curl -SL https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VER}/bin/linux/amd64/kubectl -o /binaries/kubectl \
+    && curl -SL https://get.helm.sh/helm-v${HELM_VER}-linux-amd64.tar.gz | \
+       tar -C /binaries --strip-components 1 -xvzf - linux-amd64/helm \
+    && curl -SL https://github.com/giantswarm/apptestctl/releases/download/v${APPTESTCTL_VER}/apptestctl-v${APPTESTCTL_VER}-linux-amd64.tar.gz | \
+       tar -C /binaries --strip-components 1 -xvzf - apptestctl-v${APPTESTCTL_VER}-linux-amd64/apptestctl \
+    && curl -SL https://github.com/helm/chart-testing/releases/download/v${CT_VER}/chart-testing_${CT_VER}_linux_amd64.tar.gz | \
+       tar -C /binaries -xvzf - ct etc/lintconf.yaml etc/chart_schema.yaml && mv /binaries/etc /etc/ct
+
+COPY container-entrypoint.sh /binaries
```

Comment:
Is it not too early to include it? This file is still part of the code base and its change will trigger quite ineffective build.